### ISSUE TITLE
zenny messing with camera

### ISSUE THREAD

**Issue by DamnedNForsaken** [DamnedNForsaken/MMLX#4]
Looks like when zenny are messing with the camera.
I.e. When close to enemy exploding and zenny is exploding out, they get in the way of the camera or camera boom or something and this pulls the camera really tight to megaman.
This is observed when cheap zoom is activated and far away, then run into enemy till it explodes and zenny explode out, here it will have the camera zoom in and out really quickly making for a non smooth experience.

**Comment by JJChalupnik** [comment]
Not sure about this, but the zenny behavior is very odd when they explode out. Almost looks like they explode onto a mattress, cushioning the landing before settling. Can we not use some quick form of rigid bodies? shouldn't these be physics assets, if they aren't already?

**Comment by DamnedNForsaken** [comment]
I have updated this when in the past week (before it was as you say - spawning the zenny matress like - now it has changed). Please try the latest build and tell me what you think.

**Comment by JJChalupnik** [comment]
Tthe bounce behavior feels more natural, but the objects maintain their up vector which looks really weird. Take a look at the new issue I added for zenny movement behavior. I grabbed a clip online to demonstrate what we should aim for.  I also think the pieces might be a tad too light and floaty, but the bounce looks much better. 

I am getting a strange camera thing as well, but its hard to predict when it will be reproduced. Seems to happen when I pick up the dropped zenny.
